---
title: "MOCHI enables discovery of heterogeneous interactome modules in 3D nucleome"
collection: publications
permalink: /publication/mochi
venue: '<b>Genome Research</b>'
citation: 'Dechao Tian*, <b>Ruochi Zhang*</b> (co-first author), Yang Zhang, Xiaopeng Zhu, Jian Ma. '

---  
[[PDF]](https://ruochiz.github.io/files/mochi.pdf) [[Code]](https://github.com/ma-compbio/MOCHI)


## Abstract
The composition of the cell nucleus is highly heterogeneous, with different constituents forming complex interactomes. However, the global patterns of these interwoven heterogeneous interactomes remain poorly understood. Here we focus on two different interactomes, chromatin interaction network and gene regulatory network, as a proof-of-principle, to identify heterogeneous interactome modules (HIMs), each of which represents a cluster of gene loci that are in spatial contact more frequently than expected and that are regulated by the same group of transcription factors. HIM integrates transcription factor binding and 3D genome structure to reflect “transcriptional niche” in the nucleus. We develop a new algorithm MOCHI to facilitate the discovery of HIMs based on network motif clustering in heterogeneous interactomes. By applying MOCHI to five different cell types, we found that HIMs have strong spatial preference within the nucleus and exhibit distinct functional properties. Through integrative analysis, this work demonstrates the utility of MOCHI to identify HIMs, which may provide new perspectives on the interplay between transcriptional regulation and 3D genome organization.


<p>
  <img src="https://ruochiz.github.io/images/mochi_overview.png?raw=true" alt="Photo" style="width: 500px;"/> 
</p>