---
title: "Exploiting sequence-based features for predicting enhancer-promoter interactions"
collection: publications
permalink: /publication/PEP
venue: '<b> ISMB 2017</b>, <b>Bioinformatics</b>'
citation: 'Yang Yang, <b>Ruochi Zhang</b>, Shashank Singh, Jian Ma.'
status: 'publish'
---  
[[PDF]](https://ruochiz.github.io/files/btx257.pdf) [[Code]](https://github.com/ma-compbio/PEP)


## Abstract
### Motivation
A large number of distal enhancers and proximal promoters form enhancer-promoter interactions to regulate target genes in the human genome. Although recent high-throughput genome-wide mapping approaches have allowed us to more comprehensively recognize potential enhancer-promoter interactions, it is still largely unknown whether sequence-based features alone are sufficient to predict such interactions.
### Results
Here, we develop a new computational method (named PEP) to predict enhancer-promoter interactions based on sequence-based features only, when the locations of putative enhancers and promoters in a particular cell type are given. The two modules in PEP (PEP-Motif and PEP-Word) use different but complementary feature extraction strategies to exploit sequence-based information. The results across six different cell types demonstrate that our method is effective in predicting enhancer-promoter interactions as compared to the state-of-the-art methods that use functional genomic signals. Our work demonstrates that sequence-based features alone can reliably predict enhancer-promoter interactions genome-wide, which could potentially facilitate the discovery of important sequence determinants for long-range gene regulation.

<p align="center">
  <img src="https://ruochiz.github.io/images/PEP_overview.jpg?raw=true" alt="Photo" style="width: 300px;"/> 
</p>